File: backend/Admin/src/main/java/com/example/entities/Mess.java
package com.example.entities;

import com.fasterxml.jackson.annotation.JsonIgnore;
import jakarta.persistence.*;
import lombok.*;

import java.time.LocalTime;
import java.util.List;

@Entity
@Table(name = "mess")
@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
public class Mess {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "mess_id")
    private Integer messId;

    @JsonIgnore
    @ManyToOne
    @JoinColumn(name = "user_id")
    private User userId;

    @Column(name = "mess_name", nullable = false)
    private String messName;

    @Column(name = "mess_address")
    private String messAddress;

    @Column(name = "mess_type")
    private String messType;   // Veg / Mixed / Non-Veg

    @Column(name = "lunch_open_time")
    private LocalTime lunchOpenTime;

    @Column(name = "lunch_close_time")
    private LocalTime lunchCloseTime;

    @Column(name = "dinner_open_time")
    private LocalTime dinnerOpenTime;

    @Column(name = "dinner_close_time")
    private LocalTime dinnerCloseTime;

    @ManyToOne
    @JoinColumn(name = "area_id")
    private Area areaId;

    @JsonIgnore
    @OneToMany(mappedBy = "mess", cascade = CascadeType.ALL, fetch = FetchType.LAZY)
    private List<MealMenu> mealMenus;
}

File: backend/Customer/Customer/Models/Mess.cs
using System;
using System.Collections.Generic;

namespace Customer.Models;

public partial class Mess
{
    public int UserId { get; set; }

    public int MessId { get; set; }

    public string MessName { get; set; } = null!;

    public string? MessAddress { get; set; }

    public string? MessType { get; set; }

    public TimeOnly LunchOpenTime { get; set; }

    public TimeOnly LunchCloseTime { get; set; }

    public TimeOnly DinnerOpenTime { get; set; }

    public TimeOnly DinnerCloseTime { get; set; }

    public int AreaId { get; set; }

    public virtual Area Area { get; set; } = null!;

    public virtual ICollection<FoodItemRequest> FoodItemRequests { get; set; } = new List<FoodItemRequest>();

    public virtual ICollection<MealMenu> MealMenus { get; set; } = new List<MealMenu>();

    public virtual ICollection<MonthlyPlan> MonthlyPlans { get; set; } = new List<MonthlyPlan>();

    public virtual ICollection<Rating> Ratings { get; set; } = new List<Rating>();

    public virtual User User { get; set; } = null!;

    public virtual ICollection<MessPhoto> Photos { get; set; } = new List<MessPhoto>();

}
